projects/angular/src/popover/dropdown/dropdown-item.ts

```diff
@@ -70,19 +67,10 @@ export class ClrDropdownItem implements AfterViewInit {
     return this.focusableItem.id;
   }
 
-  ngAfterViewInit() {
-    this.unlisten = this.renderer.listen(this.el.nativeElement, 'click', () => this.onDropdownItemClick());
```

Comment:
I try to remember or find a reason to have it done this way, but I can't.

Reply:
Oops, there is a failing test - it may be a clue.

Reply:
Yeah, I think handler is added in `ngAfterViewInit` to make sure it run last. I am looking into it.

Reply:
I dropped this commit. I might open a separate PR.

Reply:
I fixed it with `setTimeout`.